src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -810,21 +797,27 @@ void TLayout::layoutArpeggio(const Arpeggio* item, Arpeggio::LayoutData* ldata,
         symbolLine(font, ldata, SymId::wiggleArpeggiatoUp, SymId::wiggleArpeggiatoUp);
         // string is rotated -90 degrees
         ldata->symsBBox = font->bbox(ldata->symbols, ldata->magS);
-        ldata->setBbox(RectF(0.0, -ldata->symsBBox.x() + ldata->top, ldata->symsBBox.height(), ldata->symsBBox.width()));
+        double diff = (ldata->symsBBox.width() - (ldata->bottom - ldata->top));
+        ldata->top -= diff / 2;
+        ldata->setBbox(RectF(0.0, ldata->top, ldata->symsBBox.height(), ldata->symsBBox.width()));
     } break;
 
     case ArpeggioType::UP: {
         symbolLine(font, ldata, SymId::wiggleArpeggiatoUpArrow, SymId::wiggleArpeggiatoUp);
+        double diff = (ldata->symsBBox.width() - (ldata->bottom - ldata->top));
+        ldata->top -= diff / 2;
```

Comment:
Won't these lines need to be below the `ldata->symsBBox = font->bbox(ldata->symbols, ldata->magS);` line?